ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf

```diff
@@ -139,12 +139,13 @@ resource "aws_cloudwatch_metric_alarm" "slo_coverage_bulk_latency_99p_15m_alert"
     toset(keys(data.external.client_ssls_by_partner["coverage_latency"].result))
   )
 
-  alarm_name          = "${local.app}-${var.env}-slo-coverage-bulk-latency-99p-15m-alert-${each.key}"
-  comparison_operator = "GreaterThanThreshold"
-  evaluation_periods  = "1"
-  period              = "900"
-  extended_statistic  = "p99"
-  threshold           = local.partners.bulk[each.key].timeout_ms
+  alarm_name                            = "${local.app}-${var.env}-slo-coverage-bulk-latency-99p-15m-alert-${each.key}"
+  comparison_operator                   = "GreaterThanThreshold"
+  evaluation_periods                    = "1"
+  period                                = "900"
+  extended_statistic                    = "p99"
+  evaluate_low_sample_count_percentiles = "ignore"
```

Comment:
FYI for reviewers: Git is showing a much larger diff due to running `terraform fmt` after making these changes. Of each diff, only a single line has been added:

```
evaluate_low_sample_count_percentiles = "ignore"
```